common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -124,16 +117,11 @@ libram_of_compassion = {
 
 	stacking = no
 	indestructible = yes
-
-	diplomacy = 2
-	learning = 2
```

Comment:
I liked the diplo + learning better, but if @zumbak04 and @MemeWizard are ok with this change, I'm fine too...

Reply:
Global revolt is too big bonus, no more spaces for others.

Reply:
I know, but didn't mark the whole libram. However, as i said, since you're ok with if, I am outvoted